Prototope/DisplayLink.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  DisplayLink.swift
+//  Prototope
+//
+//  Created by Jason Brennan on 2015-08-10.
+//  Copyright (c) 2015 Khan Academy. All rights reserved.
+//
+
+import AppKit
```

Comment:
Probably worth a big comment at the top that's like, "This file is OS X only!!!111"